src/Feature/Search/SearchSubscription.re

```diff
@@ -36,6 +37,7 @@ module Make = (Config: {type action;}) => {
 
       let dispose =
         ripgrep.Ripgrep.findInFiles(
+          ~searchExclude=[],
```

Comment:
Looks like we'll need to plumb the `searchExclude` params. 

The `params` is destructured in line 26 in this file - we could add `searchExclude` to that destructuring.

Then, we can just use `~searchExclude,`, which is a short-hand for `~searchExclude=searchExclude,`